pkgs/applications/audio/plexamp/default.nix

```diff
@@ -2,13 +2,13 @@
 
 let
   pname = "plexamp";
-  version = "3.2.0";
+  version = "3.3.1";
   name = "${pname}-${version}";
```

Comment:
This line is done automatically and should be removed.

Reply:
Ah but it is used in the `installPhase` when linking  the binary.

Is there a way to access the generated version?

Reply:
keeping it around is fine